jupyterhub/app.py

```diff
@@ -290,7 +290,11 @@ def _template_paths_default(self):
         """
     ).tag(config=True)
     ip = Unicode('',
-        help="The public facing ip of the whole application (the proxy)"
+        help="""The public facing ip of the whole application (the proxy)
```

Comment:
Perhaps:

"...the whole JupyterHub application (specifically referred to as the proxy)."